source/api_cc/src/DeepPot.cc

```diff
@@ -542,6 +573,16 @@ init (const std::vector<std::string> & models, const int & gpu_rank, const std::
   }
   #endif // GOOGLE_CUDA
 
+
+  #if TENSORFLOW_USE_ROCM
+  if (gpu_num > 0) {
+      options.config.set_allow_soft_placement(true);
+      options.config.mutable_gpu_options()->set_per_process_gpu_memory_fraction(0.9);
+      options.config.mutable_gpu_options()->set_allow_growth(true);
+      hipErrcheck(hipSetDevice(gpu_rank % gpu_num));
+  }
```

Comment:
Merged with cuda GPU setting?

Reply:
if we want to merge this part of code with CUDA, maybe we should do sth to deal with hipErrcheck and cudaErrcheck

Reply:
not really, use compile condition to switch between `hipErrCheck` and `cudaErrCheck`

```cpp
#if USE_ROCM_TOOLKIT || USE_CUDA_TOOLKIT
...
#ifdef USE_ROCM_TOOLKIT
  hipErrcheck()
#elif USE_CUDA_TOOLKIT
  cudaErrcheck()
#endif
...
#endif
```